index.html

```diff
@@ -6,5 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe 3</title>
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <img src="" alt="">
+      <div>
+        <ul>
+          <li><a href= "">HOME</a></li>
```

Comment:
delete spacing after `href`

Reply:
maybe also link to `/home` etc, even though the pages don't technically exist yet

Reply:
> maybe also link e.g home: `/`, about: `/about` etc, even though the pages don't technically exist yet

above not correct, we have a single page so need anchor links `#about`